src/proto_tcp.c

```diff
@@ -690,13 +690,22 @@ int tcp_bind_listener(struct listener *listener, char *errmsg, int errlen)
 		/* we may want to try to restore the default MSS if the socket was inherited */
 		int tmpmaxseg = -1;
 		int defaultmss;
+		int v4 = listener->rx.addr.ss_family == AF_INET;
+		int mptcp = listener->rx.proto->sock_prot == IPPROTO_MPTCP;
```

Comment:
(because it is only used once, you don't really need a variable here, but up to you, I'm fine with both)